events/ready.js

```diff
@@ -9,14 +9,28 @@ module.exports = client => {
   const paneltable = new db.table('panel')
   const chalk = require('chalk')
   const config = client.config
+  const bytesConverter = require("../calculator/bytesConverter.js");
+  const percentageCalculator = require("../calculator/percentageCalculator.js");
 
+  let panelURL = config.panel.url
   let enablecs = config.botstatus.enable
   let cs = config.botstatus.text
   let stype = config.botstatus.type
   let ch = client.channels.cache.find(cn => cn.id === config.channel)
   let time = config.refreshtime
 
-  let hosturl = config.panel.url
+  //Some extra variables for the embed
+  let author = config.embed.author;
+  let authorImageURL = config.embed.authorImageURL;
+  let imageURL = config.embed.imageURL;
+  let thumbnailURL = config.embed.thumbnailURL;
+  let footerImageURL = config.embed.footerImageURL;
+  let monitorLink = config.monitorLink;
+  let adminAccountAPIKey = client.config.adminAccountAPIKey;
+  let prefix = client.config.prefix;
+  //----------------------------------------------------------------------
```

Comment:
i think you should remove this

Reply:
oops it seems like i select a bit more then the 
> //----------------------------------------------------------------------
line

Reply:
Done :)